src/plugins/intel_gpu/src/kernel_selector/kernels/reorder/reorder_kernel_base.cpp

```diff
@@ -189,6 +189,11 @@ ReorderKernelBase::DispatchData ReorderKernelBase::SetDefault(const reorder_para
         dispatchData.lws[0] = 1;
         dispatchData.lws[1] = 16;
         dispatchData.lws[2] = 1;
+    } else if (input_l == DataLayout::ybfx) {
+        dispatchData.gws[2] = input.Batch().v;
+        dispatchData.gws[1] = input.Feature().v;
+        dispatchData.gws[0] = input.Y().v*input.X().v;
+        dispatchData.lws = {1, 1, 1};
```

Comment:
This workgroup might not provide optimal performance, we may consider optimizing it in the future

Reply:
ok